FESS-GUI/commoninterfaceselector.ui

```diff
@@ -219,7 +219,7 @@
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="currentText">
-            <string>8</string>
```

Comment:
The default number of data bits for most serial devices is 8. Thats why this is set as 8.